internal/storagenode/logstream/executor.go

```diff
@@ -348,8 +348,11 @@ func (lse *Executor) Report(_ context.Context) (snpb.LogStreamUncommitReport, er
 	atomic.AddInt64(&lse.inflight, 1)
 	defer atomic.AddInt64(&lse.inflight, -1)
 
-	if lse.esm.load() == executorStateClosed {
+	switch lse.esm.load() {
```

Comment:
Is there any chance that `lse.esm.load` and `lse.lsc.reportCommitBase` will be in learning state between executions so incorrect information will be read?